test/tasks/removeTokens.test.ts

```diff
@@ -7,13 +7,14 @@ import { ethers } from 'hardhat';
 import sinon from 'sinon';
 import { removeTokens } from '../../tasks/removeTokens';
 import { stubReadFileSync } from './utils';
+import { AllowedTokensArgs } from 'tasks/allowTokens';
 
 use(chaiAsPromised);
 
 describe('Remove Tokens Script', function () {
```

Comment:
We are missing some testing, ie:

- should remove a list of tokens and verifiers
- should remove a single token and verifiers
- should remove a list of tokens and a single verifier